package.json

```diff
@@ -1678,7 +1679,7 @@
     "@types/webpack-sources": "^0.1.4",
     "@types/xml2js": "^0.4.11",
     "@types/yargs": "^15.0.0",
-    "@types/yauzl": "^2.9.1",
+    "@types/yauzl": "^2.10.3",
```

Comment:
nit: seems odd that the types are v2 and the library v3. Maybe these are not needed anymore?

Reply:
Thanks for noting - they're still needed.  They missing changes aren't too extensive fortunately, it looks like there are missing definitions for a few methods we aren't using.


https://github.com/DefinitelyTyped/DefinitelyTyped/tree/ffe46d9382e765fc0f54530b4653e57e6ef0921c/types/yauzl
https://github.com/thejoshwolfe/yauzl?tab=readme-ov-file#change-history